docs/release-template.md

```diff
@@ -80,4 +82,8 @@
 
 ## TPC-H connector
 
+## TPC-DS connector
```

Comment:
Seems like in the docs and such we actually use TPCDS .. if we make it consistent.. whats your preference @martint

Reply:
https://github.com/trinodb/trino/pull/24082